### DISCUSSION TITLE
Boxed.Mapping 4.0.0 release notes?

### DISCUSSION THREAD

**Discussion by nmeyering** [ASP-NET-MVC-Boilerplate/Framework#144] [Q&A]
Boxed.Mapping 4.0.0 seems to be on NuGet but I don't see a tag or release notes on GitHub. 
Which commit is it, what changed?

https://www.nuget.org/packages/Boxed.Mapping/4.0.0

**Comment by RehanSaeed** [comment]
Apologies, forgot to write the notes:

https://github.com/Dotnet-Boxed/Framework/releases/tag/Boxed.Mapping.4.0.0
